packages/components/managed-data-table/src/data-table-settings/droppable-panel/droppable-panel.styles.js

```diff
@@ -0,0 +1,13 @@
+import styled from '@emotion/styled';
+import { customProperties as vars } from '@commercetools-uikit/design-system';
+
+const TagContainerEditable = styled.div`
+  background-color: ${vars.backgroundColorForInput};
+  border: ${vars.borderRadius1} solid ${vars.colorNeutral60};
+  border-radius: ${vars.borderRadius6};
+  padding: ${vars.spacingS};
+  height: ${vars.constraintM};
+  overflow: auto;
+`;
+
+export default TagContainerEditable;
```

Comment:
The file name is wrong, you're exporting a react component here. Rename the file to `droppable-panel.js`, not `.styles.js`

Reply:
This is applying styles over a component a div. I understand this is the way things are being done on the UIKit. The only difference is that I'm only exporting one component so it's the default. I can disable the rule so it allows 
```
/* eslint-disable import/prefer-default-export */
export { TagContainerEditable };
```

That way it would be working as 
- `packages/components/managed-data-table/src/data-table-settings/data-table-settings-panel.styles.js`
- `packages/components/data-table/src/data-table.styles.js`
- `packages/components/data-table/src/header-cell.styles.js`
- `...`

Am I not understanding what you're saying?

Reply:
No the styles files are used to define css (in-js) styles, they export the emotion styles.

Here you have a styled component, which is a normal React component and it should be treated as such.

Does it make sense?

Reply:
Sorry @emmenko, it's my first time working with emotion and I'm not quite sure I see what you mean. Here we are adding css code to a div
```
const TagContainerEditable = styled.div`
  background-color: ${vars.backgroundColorForInput};
  border: ${vars.borderRadius1} solid ${vars.colorNeutral60};
  border-radius: ${vars.borderRadius6};
  padding: ${vars.spacingS};
  height: ${vars.constraintM};
  overflow: auto;
`;
```
This means this is a styled component?  I was trying to base my code on the `DataTable` component, so I've assumed this was the way to go [data-table.styles.js](https://github.com/commercetools/ui-kit/blob/master/packages/components/data-table/src/data-table.styles.js)

Should then I renamed the file as `tag-container-editable.js` ?

Reply:
Yes, this is a styled component, which is a normal React component. The `data-table.styles.js` is probably not a good example because it mixes styles with react components.

If this file `droppable-panel.styles.js` only exports one component, I would just rename the file to match the component name.